packages/namekit-react/src/index.ts

```diff
@@ -14,3 +14,4 @@ export * from "./components/Input";
 export * from "./components/ENSInput";
 export * from "./components/Link";
 export * from "./components/Tooltip";
+export * from "./components/CurrencySymbol/CurrencySymbol";
```

Comment:
@FrancoAguzzi Please create changesets whenever a PR impacts one of the packages we publish to NPM.